Apps/Medication/src/Models/Prescription.cs

```diff
@@ -0,0 +1,93 @@
+﻿//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.MedicationService.Models
+{
+    /// <summary>
+    /// The patient data model.
+    /// </summary>
+    public class Prescription
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Prescription"/> class.
+        /// </summary>
+        public Prescription()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Prescription"/> class.
+        /// </summary>
+        /// <param name="phn">The patient's personal health number to scope the medication statement.</param>
+        public Prescription(string phn)
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets the Drug Identification Number for the Medication.
+        /// </summary>
+        public string DIN { get; set; }
+
+        /// <summary>
+        /// Gets or sets the brand name of the medication.
+        /// </summary>
+        public string BrandName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the common or generic name of the medication.
+        /// </summary>
+        public string GenericName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the quantity for the prescription.
+        /// </summary>
+        public int Quantity { get; set; }
```

Comment:
Would this be number of tablets or something like that?

Reply:
The specifications were not clear on this, we need more information.